setup.py

```diff
@@ -70,7 +70,15 @@
         library_dirs=default_lib_dirs,
         include_dirs=default_include_dirs,
         define_macros=define_macros,
-        )
+        ),
+    Extension(
+        "flint.flint_base.flint_context",
+        ["src/flint/flint_base/flint_context.pyx"],
+        libraries=libraries,
+        library_dirs=default_lib_dirs,
+        include_dirs=default_include_dirs,
+        define_macros=define_macros,
+    ),
```

Comment:
It would be better to factor this out like:
```
ext_files = [
    ('"flint._flint", ["src/flint/pyflint.pyx"]),
    ...
]

ext_options = {
    'libraries': libraries,
    ...
}

ext_modules = []
for mod_name, src_files in ext_files.items():
    ext = Extension(mod_name, src_files, **ext_options)
    ext_modules.append(ext)
```
Then it is easier to add to the `ext_files` list to make more extension modules and we don't need to duplicate the arguments in `ext_options`.

Reply:
Agreed, addressed in commit https://github.com/flintlib/python-flint/pull/61/commits/f7c3bfc5f36d724a7c38b6a9a9ec6dfb0ff57bd9